src/test/java/core/basesyntax/service/impl/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReaderService;
+import java.nio.file.Path;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderServiceImplTest {
+    //1. file path is wrong another from src/test/resources/
+    //2. file path is wrong src/test/resources/no file
+    private static CsvFileReaderService fileReaderService;
+    private static final String VALID_FILE_PATH = "src/test/resources/file.csv";
+
+    @BeforeClass
+    public static void setUp() {
+        fileReaderService = new CsvFileReaderServiceImpl();
+    }
+
+    @Test
+    public void fileReader_filePathRight_Ok() {
+        Assert.assertEquals(VALID_FILE_PATH, "src/test/resources/file.csv");
+    }

```

Comment:
Looks like in this test you compare constant with its value.
You initialize this constant with this value by yourself, so this test is unnecessary


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">